src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,5 +9,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_DECK = new Prefix("d/");
```

Comment:
May I ask what is the exact syntax of `addDeck` command. Is it `addDeck d/deck name` ?